### PR TITLE
feat: add support for telemetry

### PR DESCRIPTION
**Pull request by asalkeld**
part of https://github.com/nitrictech/nitric/issues/329

### CODE REVIEW

File: pkg/runtime/wrapper.go [hunk 1]
```diff
@@ -39,6 +43,61 @@ CMD [%s]
...
 `
 
+const wrapperDockerFileWithOTel = `
+ARG BASE_IMAGE
+
+FROM ${BASE_IMAGE}
+
+RUN apk update \
+  &&   apk add --no-cache ca-certificates wget shadow \
+  &&   update-ca-certificates
```
Comment (tjholm) [inline pkg/runtime/wrapper.go]: Is the base image always alpine?

**Review by tjholm — CHANGES_REQUESTED** [review]
Looks pretty good, need more time to go over this, but first thing that pops up is an assumption about base image package managers that could limit future functionality.

File: pkg/runtime/wrapper.go (hunk 1 above)
Comment (asalkeld) [reply pkg/runtime/wrapper.go]: I have moved these changes into the base images

File: pkg/runtime/wrapper.go [hunk 2]
```diff
@@ -30,15 +36,72 @@ ARG BASE_IMAGE
...
+FROM ${BASE_IMAGE}
+
+ARG MEMBRANE_URI
+ARG MEMBRANE_VERSION
+
+ENV MEMBRANE_VERSION ${MEMBRANE_VERSION}
+
+RUN apk add --no-cache wget && \
+    wget -q ${MEMBRANE_URI} -O /bin/membrane && \
+    chmod +x-rw /bin/membrane
```
Comment (tjholm) [inline pkg/runtime/wrapper.go]: Still using `apk` here

**Review by tjholm — CHANGES_REQUESTED** [review]

File: pkg/runtime/wrapper.go (hunk 2 above)
Comment (asalkeld) [reply pkg/runtime/wrapper.go]: sorted

**Review by tjholm — COMMENTED** [review]
@asalkeld are the membrane changes required before this gets released? I'm assuming this can be done independently but wanted to check.

**Comment by asalkeld** [comment]
> 

either first should be fine.

**Review by tjholm — APPROVED** [review]

**Pull request merged**